.github/workflows/gke.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
   installation-and-connectivity:
     if: ${{ github.repository == 'cilium/cilium-cli' }}
     runs-on: ubuntu-latest
-    timeout-minutes: 30
+    timeout-minutes: 45
```

Comment:
commit message says increase timeout for GKE and External Workload, but only gke.yaml changed.

Reply:
thanks, external workload was done in parrallel and got merged to master, I will update commit message along with temp commit later.